Dockerfile-nobuild

```diff
@@ -0,0 +1,4 @@
+FROM opensuse/leap:15.0
+RUN zypper -n in system-user-nobody
+USER nobody
+COPY binaries/cf-operator /usr/local/bin/cf-operator
```

Comment:
I guess it's not possible to do this in the same dockerfile.  `RUN if` maybe?

Reply:
I don't think what you mean is possible.